packages/blade/src/components/Checkbox/Checkbox.tsx

```diff
@@ -50,7 +50,7 @@ type CheckboxProps = {
   /**
    * Sets the label of the checkbox
    */
-  children: React.ReactNode;
+  children?: React.ReactNode;
```

Comment:
why is it `React.ReactNode` here and in radio `StringChildrenType`?

Reply:
No idea actually, seems like a miss. But changing this might cause breaking change.